eZ/Publish/API/Repository/SearchService.php

```diff
@@ -96,4 +97,13 @@ public function suggest($prefix, $fieldPaths = array(), $limit = 10, Criterion $
      * @return \eZ\Publish\API\Repository\Values\Content\Search\SearchResult
      */
     public function findLocations(LocationQuery $query, array $languageFilter = array(), $filterOnUserPermissions = true);
+
+    /**
+     * Get SortClause objects built from $location's sort options.
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Query\SortClause[]
+     */
+    public function getSortClauseFromLocation(Location $location);
```

Comment:
Is it so ok? https://github.com/ezsystems/ezpublish-kernel/pull/1809/commits/a543d92b1233f3b08f54a1e8de1a620fe02168b2